doc/services/portability/posix/conformance/index.rst

```diff
@@ -0,0 +1,147 @@
+.. _posix_conformance:
+
+POSIX Conformance
+#################
+
+As per `IEEE 1003.1-2017`, this section details Zephyr's POSIX conformance.
+
+.. _posix_undefined_behaviour:
+
+.. note::
+   As per POSIX 1003.13, single process mode is supported directly by and PSE51 and PSE52 profiles.
+   While Zephyr includes support for many features found in PSE53, PSE53 itself requires supporting
+   multiple processes. Since supporting multiple processes is beyond the scope of Zephyr's current
+   design, some features requiring multi-process capabilities may exhibit undefined behaviour,
+   which we denote with the † (obelus) symbol.
+
+.. _posix_system_interfaces:
+
+POSIX System Interfaces
+=======================
+
+The following have values greater than -1 in Zephyr, conformant with the POSIX specification.
+
+.. csv-table:: POSIX System Interfaces (>-1)
+   :header: Symbol, Support, Remarks
+   :widths: 50, 10, 50
+
+    _POSIX_CHOWN_RESTRICTED, 0,
+    _POSIX_NO_TRUNC, 0,
+    _POSIX_VDISABLE, 0,
+
+The following should be valued greater than zero in Zephyr, in order to be strictly conformant
```

Comment:
I would not write "should" in this type of documentation. A reader can understand that this is implying any of "that it is supported, but due to a bug these defines are not set properly", "that zephyr does not support this", "that zephyr intends to support it,..

Reply:
Fixed!